src/app/@ansyn/plugins/openlayers/plugins/monitor/monitor.plugin.ts

```diff
@@ -43,12 +56,14 @@ export class MonitorPlugin extends BaseImageryPlugin {
 
 	onResetView() {
 		return <Observable<boolean>>super.onResetView()
-			.pipe(tap(this.monitorSource.bind(this)));
+			// .pipe(tap(this.monitorSource.bind(this)))
```

Comment:
The monitor plugin now cannot be started from OnResetView(), like it used to, because the monitor needs to work _before_ resetView() is resolved. Anyway after the refactor that I did to the monitor, most of it is not in the plugin, and OnResetView() is no longer used.